projects/greenhouse/index.js

```diff
@@ -0,0 +1,73 @@
+const sdk = require("@defillama/sdk");
+const erc20 = require("../helper/abis/erc20.json");
+const abi = require("./abi.json");
+const { unwrapUniswapLPs } = require("../helper/unwrapLPs");
+const { transformPolygonAddress } = require("../helper/portedTokens");
+
+const MASTERCHEF_CONTRACT = "0xbD40a260Ddd78287ddA4C4ede5880505a9fEdF9a";
```

Comment:
@waynebruce0x  is `addFundsInMasterChef` applicable for this adapter?

Reply:
yesss I think that's a good suggestion, thanks g1nt0ki